search/search.model.lkml

```diff
@@ -16,14 +16,14 @@ explore: +desktop_search_counts {
 explore: desktop_search_alert_latest_daily {}
 
 explore: business_development_core_search_users_monthly {
-  view_name: bizdev_search_core_users
+  view_name: desktop_mobile_search_clients_monthly
 
-  aggregate_table: rollup__bizdev_search_core_users_last_month {
+  aggregate_table: rollup__desktop_mobile_search_clients_monthly_last_month {
     query: {
-      dimensions: [bizdev_search_core_users.days_of_use_bucket, bizdev_search_core_users.ad_click_bucket, bizdev_search_core_users.country, bizdev_search_core_users.normalized_engine, bizdev_search_core_users.submission_month]
-      measures: [bizdev_search_core_users.ad_clicks, bizdev_search_core_users.clients, bizdev_search_core_users.searches]
+      dimensions: [desktop_mobile_search_clients_monthly.days_of_use_bucket, desktop_mobile_search_clients_monthly.ad_click_bucket, desktop_mobile_search_clients_monthly.country, desktop_mobile_search_clients_monthly.normalized_engine, desktop_mobile_search_clients_monthly.submission_month]
+      measures: [desktop_mobile_search_clients_monthly.total_ad_clicks, desktop_mobile_search_clients_monthly.clients, desktop_mobile_search_clients_monthly.total_searches]
```

Comment:
```suggestion
      measures: [desktop_mobile_search_clients_monthly.total_ad_clicks, desktop_mobile_search_clients_monthly.approx_clients, desktop_mobile_search_clients_monthly.total_searches]
```
I'd suggest using `approx_clients` instead of `clients` in aggregate tables unless the queries that people are interested in are for that specific set of dimensions. This won't be able to optimize for eg.num clients per month per days of use bucket (since it doesn't include ad click bucket)